pkgs/leak_tracker/lib/src/leak_tracking/_object_record.dart

```diff
@@ -63,7 +63,10 @@ class ObjectRecord {
   DateTime? _gcedTime;
   int? _gcedGcCount;
   void setGCed(int gcCount, DateTime time) {
-    if (_gcedGcCount != null) throw Exception('The object $code GCed twice');
+    // Normally it should not happen, but sometimes finalizer is called twice.
+    // To repro, update next line to throw exception and run flutter tests with
+    // the updated leak_tracker.
+    if (_gcedGcCount != null) return;
```

Comment:
Please add a TODO with a bug tracking this. The Dart VM really should not be calling finalizers twice.
Do we have a standalone repro so that @rmacnak-google and others on the Dart team can reproduce?

Reply:
Done: https://github.com/dart-lang/sdk/issues/55330